sdk/core/azure-core/src/main/java/com/azure/core/http/HttpHeaders.java

```diff
@@ -15,6 +15,17 @@
  * A collection of headers on an HTTP request or response.
  */
 public class HttpHeaders implements Iterable<HttpHeader> {
+
+    /**
+     * The header which will have retry after value in milliseconds.
+     */
+    public static final String AZURE_RETRY_AFTER_MS_HEADER = "retry-after-ms";
```

Comment:
I think these should be moved to the `RetryPolicy` itself, that would make them more discoverable.